README.md

```diff
@@ -550,6 +558,68 @@ Klaviyo.unregisterFromInAppForms()
 
 Note that after unregistering, the next call to `registerForInAppForms()` will be considered a new session by the SDK.
 
+## Geofencing
```

Comment:
This sounds good to me. One nit - Any chance to skim some stuff to make this more of an pointed integration guide? For example, would devs care about, 

```
Once registered, the SDK will:
1. Monitor for required location permissions to be granted
2. Fetch geofence data from your Klaviyo account once permission is granted
3. Register geofences with Android's location services
4. Automatically create events when users enter or exit geofences
5. Restore geofences after device reboot (if permissions are still granted)
```

Reply:
I think you're right though, that stuff could be further refined.